### DISCUSSION TITLE
Deploy different files of the same repository

### DISCUSSION THREAD

**Discussion by HugoAlbertBonet** [vercel/community#2127] [Help] [answered]
### Summary

I have the following repository: https://github.com/HugoAlbertBonet/wedding

Where I have my main.ts and my index.html, which are working perfectly with vercel.
However, I have another secondary.ts and transmit.html which I expected to work at the same time (they worked locally) by visiting the same url but changing /index.html for /transmit.html but it is not working.

Besides, they post media files in the folder /src/audio where app.py is supposed to receive those audios (locally I accessed it as http://localhost:5050 but now I do not know how to do it so I am not sure if the app.py is initialized).

Can I solve those problems in any way or do I have to create another two independent deployments?

**Comment by amyegan** [comment, marked as answer]
Thanks for sharing the repo, @HugoAlbertBonet! Deploying it all together is possible, but I'm not familiar enough with Python + Vite to be able to say exactly what the solution would be in this case. 

It looks like `vite build` isn't including the all of your HTML files in the `dist` output. You may need to add a [Vite config](https://vitejs.dev/config/#configuring-vite) file to get the build output to include the files you want.

For the Python files, you might need to change your file structure or include a [Vercel config](https://vercel.com/docs/concepts/projects/project-configuration#functions) to define serverless functions. You can learn more about running Python code in the [Python Runtime](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/python) documentation. There are also some [Python starter templates](https://vercel.com/templates?search=python) available that show recommended setups with example code.

**Reply by HugoAlbertBonet** [reply]
Thank you so much. However, it is my first time using vite so I do not know what I have to add to the vite config, could you help me with that please?

**Reply by amyegan** [reply]
I can't offer Vite support. I'm not an expert on it and don't want to give you bad advice! But you should be able to get some help with configuration over in the [Vite repo discussions](https://github.com/vitejs/vite/discussions) or [Vite Discord](https://chat.vitejs.dev/)
